zone/special_attacks.cpp

```diff
@@ -1819,7 +1819,8 @@ void NPC::DoClassAttacks(Mob *target) {
 		target->GetBodyType() != BT_Undead &&
 		taunt_time &&
 		type_of_pet &&
-		type_of_pet != petTargetLock
+		type_of_pet != petTargetLock &&
+		DistanceSquared(this->GetPosition(), target->GetPosition()) <= (RuleI(Pets, PetTauntRange)*RuleI(Pets, PetTauntRange))
```

Comment:
No need for `this`. Can also just create a variable to hold pet taunt range.
```cpp
const int pet_taunt_range = (RuleI(Pets, PetTauntRange) * RuleI(Pets, PetTauntRange));
```

Reply:
Will remove `this` but is a variable needed when its only referenced once? e.g. is that general best practice going forward?

Reply:
We can leave it as is.